envergo/pages/views.py

```diff
@@ -16,7 +17,12 @@
 
 
 class HomeView(MoulinetteMixin, FormView):
-    template_name = "pages/home.html"
+    def get_template_names(self):
+        current_site = get_current_site(self.request)
+        home = "pages/home.html"
```

Comment:
Peut-être qu'on peut prévenir une future cascade de `ifs` ?

```suggestion
        home = f"pages/home_{current_site}.html"
```